components/raftstore-v2/src/operation/command/admin/merge/commit.rs

```diff
@@ -319,7 +319,7 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
                 region
             );
             assert!(!self.storage().has_dirty_data());
-            if self.is_leader() {
+            if self.is_leader() && !self.leader_transferring() {
```

Comment:
L353's log is inaccurate now. It's better to know if it's because of !is_leader() or leader_transferring()

Reply:
So the AskCommitMerge would be ignored and the source region will send again later ?

Reply:
> So the AskCommitMerge would be ignored and the source region will send it again later ?

Source region on all nodes will send AskCommitMerge periodically, regardless of whether the source region is leader or target region is leadeer.